docs/reference/announcements.md

```diff
@@ -104,6 +104,10 @@ This release contains the following limitations:
     - **Description:** OpenSearch support in Optimize is limited to data import and the raw data report.
     - **Reference:** n/a
     - **Mitigation:** Optimize can be installed and used in production with limited reporting functionality. Optimize imports all process data generated by Zeebe. All reporting functionality as described in the docs will be delivered with upcoming patches.
+- In **Console `8.5.x`**
```

Comment:
Tweak/nitpick: This is the preview of 8.6 docs, but we often wait to make changes until it's available in an alpha. So, I could see this staying as is and getting removed when it's no longer a limitation (if it will be delivered with 8.6.0 as noted in the public issue).

Reply:
If it's no longer a limitation in 8.6,  it will still be a limitation for 8.5. Do you think it will be confusing to people because they are on the preview  despite being underneath the `Camunda 8.5` heading?

Reply:
Oh. Ha! That's what I get for not looking at the full page. Ignore me.